misc.go

```diff
@@ -408,6 +408,7 @@ var tokenMap = map[string]int{
 	"QUERIES":                  queries,
 	"QUICK":                    quick,
 	"SHARD_ROW_ID_BITS":        shardRowIDBits,
+	"PRE_SPLIT_REGIONS":        preSplitResions,
```

Comment:
```suggestion
	"PRE_SPLIT_REGIONS":        preSplitRegions,
```

Fix the PR title too.